tmva/sofie/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 1995-2021, Rene Brun and Fons Rademakers.
+# All rights reserved.
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+############################################################################
+# CMakeLists.txt file for building TMVA SOFIE package
+############################################################################
+#Author: Sitong An, Lorenzo Moneta 10/03/2021
+
+#sofie is built only if protobuf is found
+
+include_directories(SYSTEM ${Protobuf_INCLUDE_DIRS} ${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
Could you please change line 14 (I didn't test these line so please test it) and move it at the end of CMakeLists file:

```
target_include_directories(ROOTTMVASofie PUBLIC
  $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>)
target_include_directories(ROOTTMVASofie PUBLIC
  ${Protobuf_INCLUDE_DIRS})
set_target_properties(ROOTTMVASofie PROPERTIES
  POSITION_INDEPENDENT_CODE TRUE)
```

Reply:
Done and tested